xdsl/dialects/onnx.py

```diff
@@ -255,6 +255,83 @@ def verify_(self) -> None:
             )
 
 
+@irdl_op_definition
+class Reshape(IRDLOperation):
```

Comment:
is there any documentation we could add here as a doc comment?

Reply:
ill add the doc in now